tool/setup.sh

```diff
@@ -6,3 +6,4 @@ dart pub global activate melos 3.1.0
 dart pub global activate fvm 2.4.1
 echo "y" | fvm install
 melos exec --concurrency=1 flutter pub get
+dart run husky install
```

Comment:
does it make sense to use `fvm dart ...`?

Reply:
I don't think it matters since it's only for the workspace package and that has a very loose constraint. Also  we are using the direct dart command earlier in the script so it wouldn't fail here